tutorial/task_presentation.md

```diff
@@ -0,0 +1,131 @@
+# task-parallelism with Pyccel
+
+## A task
+
+Parallel program is viewed as a collection of tasks that communicate by sending
+messages to each other through dependencies.
+A task consists of an executable unit or a unit of computation (think of it as a program),
+together with its local memory and a collection of I/O ports. that can/should execute in
+parallel with other tasks.
+
+## Context
+
+A problem can be borken into multiple parts each part can be represented as a task, each
+task is a separate unit of work that can be independant or may take some dependencies from other
+tasks. task with dependencies may only start when all antecedents have completed. 
+So in general the dependencies between tasks can be represented as a directed acyclic graph
+where tasks form the vertices edge are the dependencies between task.
+
+## Usage:
+Using the decorator task allows pyccel to recognize that the current function will
```

Comment:
I think you should highlight the decorator name in some way (code snippet/italics/bold/etc)